tests/webapp/api/test_jobs_api.py

```diff
@@ -4,16 +4,17 @@
 
 from treeherder.model.models import (ExclusionProfile,
                                      JobExclusion)
+from treeherder.webapp.api.jobs import JobsViewSet
 
 
-def test_job_list(webapp, eleven_jobs_stored, jm):
+def test_job_list(webapp, eleven_jobs_stored, test_repository):
     """
     test retrieving a list of ten json blobs from the jobs-list
     endpoint.
     """
     resp = webapp.get(
         reverse("jobs-list",
-                kwargs={"project": jm.project})
+                kwargs={"project": test_repository.name})
```

Comment:
I'm confused why this is using `test_repository.name` whereas the other tests are still using `jm.project`?